packages/server/database/types/MeetingTeamPrompt.ts

```diff
@@ -57,8 +57,7 @@ export default class MeetingTeamPrompt extends Meeting {
       phases,
       facilitatorUserId,
       meetingType: 'teamPrompt',
-      //TODO: use client timezone here (requires sending it from the client and passing it via gql context most likely)
-      name: createTeamPromptTitle(name || 'Standup', new Date(), 'UTC'),
+      name,
```

Comment:
+1 We should make `name` on the `Input` interface required.

Reply:
It'd require frontend changes, as currently, we don't require users to provide a meeting series name. We could always use `Standup` by default but this seems to be a different question. I'll merge this one, we'll can discuss further improvements before jumping into impl